src/ucp/wireup/wireup_cm.h

```diff
@@ -19,8 +19,7 @@ typedef struct ucp_cm_client_connect_progress_arg {
 } ucp_cm_client_connect_progress_arg_t;
 
 
-unsigned ucp_cm_ep_init_flags(const ucp_worker_h worker,
-                              const ucp_ep_params_t *params);
+unsigned ucp_cm_ep_init_flags(const ucp_worker_h worker, const ucp_ep_params_t *params);
```

Comment:
can you please just revert as it was before?

Reply:
done